src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/clangcl.pe.cpp.codeview.exe.sarif

```diff
@@ -381,11 +381,12 @@
         {
           "ruleId": "BA2024",
           "ruleIndex": 16,
+          "kind": "pass",
```

Comment:
>pass [](http://example.com/codeflow?start=19&length=4)

In the baseline changes,
all built with debug (including this one with -Mtd)
was failing now will pass.